.github/workflows/deploy-nextjs-app-pr.yaml

```diff
@@ -0,0 +1,46 @@
+name: Deploy website from PR
+
+on:
+  pull_request:
+    branches: [main]
+    types: [labeled, edited]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.label.name == 'PR_DEPLOY' }}
+    defaults:
+      run:
+        working-directory: ./apps/next
+    env:
+      NODE_ENV: ci
+      RAILWAY_SERVICE_NAME: ${{ vars.RAILWAY_SERVICE_NAME }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Use Node.js 20
+        uses: actions/setup-node@v4
+        with:
+          node-version: '>=20'
+
+      - name: Use Bun
+        uses: oven-sh/setup-bun@v1
+        with:
+          bun-version: 1.1.3
+
+      - name: Install packages
+        run: bun install --frozen-lockfile
+
+      - name: Push DB changes
+        run: bun run ci:db:push
+        env:
+          DRIZZLE_DISABLE_STRICT_PUSHES: ${{ vars.DRIZZLE_DISABLE_STRICT_PUSHES }}
+          POSTGRES_CA_DATA: ${{ secrets.POSTGRES_CA_DATA_DEV }}
+          POSTGRES_URL: ${{ secrets.POSTGRES_URL_DEV }}
```

Comment:
so this looks like the only difference with the dev GHA. What's the point of having the dev environment GHA? Part of me thinks we should leave the dev environment alone from a GHA perspective, and use it as something we deploy to locally (and if we step on toes we can create dev-raf, dev-folusho, etc.)

Reply:
Yeah sure. I'll delete the GHA for dev. LMK if you want me to leave it but remove the on-push-to-main-branch trigger.